.github/workflows/build.yml

```
@@ -0,0 +1,52 @@
+name: Build and push to build branch.
+
+on:
+    push:
+        # `try/build-branch` only here for testing, can remove once PR is merged.
+        branches: [trunk, try/build-branch]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
+
+            - name: Install NodeJS
+              uses: actions/setup-node@5b52f097d36d4b0b2f94ed6de710023fbb8b2236 # v3.1.0
+              with:
+                  node-version-file: '.nvmrc'
+                  cache: 'yarn'
+
+            - name: Install all dependencies
+              run: |
+                  composer install
+                  yarn
+
+            - name: Build
+              run: yarn workspaces run build
+
+            - name: Trim the repo down to just the theme
+              run: |
+                  cp -R source/wp-content/themes/wporg-parent-2021 theme
+                  rm -rf theme/node_modules
+                  git rm -r .
+                  mv theme/* .
+                  rm -r theme
```

Looking at https://github.com/WordPress/wporg-parent-2021/tree/build I can see a `node_modules`, `vendor` and `source/wp-content` folders still.

This seems to clear it up entirely though to just the contents of the wporg-parent-2021:
```suggestion
            - name: Trim the repo down to just the theme
              run: |
                  rm -rf source/wp-content/themes/wporg-parent-2021/node_modules
                  mv source/wp-content/themes/wporg-parent-2021 $RUNNER_TEMP
                  git rm -rf .
                  rm -rf *
                  mv $RUNNER_TEMP/wporg-parent-2021/* .
```

By using `$RUNNER_TEMP` we can move the files we want to keep outside of the checkout. RUNNER_TEMP is cleared before/after each task.

By using `rm` after `git rm` we can remove the non-tracked files (eg node_modules and vendor). I'm not sure why `source/wp-content` was remaining, but this cleans that up too.

---

Thanks! I've applied your changes and it looks better here too 💯